CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+---
+
+Thank you for your interest in contributing to the P4 Compiler tutorials repository! Your contributions are important and will help to improve the project for everyone. Before you begin, please consider the guidelines below.
+
+## Getting Started
+
+- Make sure you have [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) installed on your machine.
+- Fork the repository and clone it locally.
+  
+  ```
+  git clone https://github.com/p4lang/tutorials.git
```

Comment:
Shouldn't `p4lang` in that URL be the user's Github id, if they are cloning their forked version?

Reply:
No, p4lang is correct here.

Reply:
The earlier instructions say to fork the repository.  When a Github user forks a repository, it creates a new repository with the URL https://github.com/$USER/tutorials, and if they want to create a branch in it and create a PR, that is the repo they should create the branch in and make the commit.

If they make the branch in the original repo and try to push it, typically it will fail because they will not have push permissions on it.

Reply:
Okay, I get what you mean. Thanks for making this clear.